modules/oe_whitelabel_extra_project/oe_whitelabel_extra_project.module

```diff
@@ -118,14 +118,24 @@ function _oe_whitelabel_extra_project_preprocess_featured_media(&$variables) {
 function _oe_whitelabel_extra_project_preprocess_inpage_nav(array &$variables): void {
   /** @var \Drupal\node\NodeInterface $node */
   $node = $variables['node'];
-
   $variables['inpage_navigation_links'] = [];
+  $variables['project_details_group'] = FALSE;
+
+  // Get field groups.
+  foreach ($variables['elements']['#fieldgroups'] as $group) {
```

Comment:
this is not completely correct, you should target only the groups that are part of the details group

Reply:
I looked deeper into the issue, the problem is that the html element group prints the title even if the group is empty. So remove this solution, hide the title from the group, implement the old solution but add also striptags before trim. Then the old code works, but you have to add the "Project details" title in the template.